ddtrace/constants.py

```diff
@@ -44,7 +44,7 @@
 MANUAL_KEEP_KEY = "manual.keep"
 
 
-ERROR_MSG = "error.msg"  # a string representing the error message
+ERROR_MESSAGE = "error.message"  # a string representing the error message
```

Comment:
We cannot change this name since it is part of the public API.

We need to keep the variable name as `ERROR_MSG`... or we need to have it aliased.

```python
ERROR_MSG = ERROR_MESSAGE  # Add deprecation warning?
```